### PR TITLE
fix: Fix proc-macro slow test

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: crates/rust-analyzer/tests/slow-tests/main.rs [hunk 1]
```diff
@@ -916,7 +927,7 @@ pub fn foo(_input: TokenStream) -> TokenStream {
         },
         "procMacro": {
             "enable": true,
-            "server": PathBuf::from(env!("CARGO_BIN_EXE_rust-analyzer")),
```
Comment (lnicola) [inline crates/rust-analyzer/tests/slow-tests/main.rs:919]: Right, noticed this recently and was wondering how it can work..

Comment (Veykril) [reply crates/rust-analyzer/tests/slow-tests/main.rs:919]: I'm more surprised that I didn't test this locally when I removed that rust-analyzer command 😰 I could swear I did, though maybe I didn't set the env var for slowt tests correctly

**Pull request merged**